scripts/build.ps1

```diff
@@ -161,9 +161,9 @@ function Restore-Package
     $dotnetExe = Get-DotNetPath
 
     Write-Log ".. .. Restore-Package: Source: $TPB_Solution"
-    & $dotnetExe restore $TPB_Solution --packages $env:TP_PACKAGES_DIR -v:minimal -warnaserror -p:Version=$TPB_Version
+    & $dotnetExe restore $TPB_Solution --packages $env:TP_PACKAGES_DIR -v:minimal -warnaserror:false -p:Version=$TPB_Version
```

Comment:
Why are we setting `-warnaserror` as false?

Reply:
since we are downgrading Xml.ReaderWriter, the dotnet compiler throws warning, as the one that comes as part of NetStandard meta package is of higher version. Although this does not fail the build, but these warnings show up in red, so disabling marking warning as errors